services/ui-auth/serverless.yml

```diff
@@ -129,6 +148,7 @@ resources:
           - email
           - openid
           - profile
+          - phone
```

Comment:
is this going to affect existing users in any way, or just pull it through?

Reply:
phone slipped through my copy paste. removing it now. ty.

No placeholder stuff in UI, gets handled from cognito/handshake.

intentionally leaving SSM out in 1st iteration to ensure getting same SAML behavior. Will add SSM param in after initial deploy and validate OIDC.

Reply:
the SSM for SAML exists, and 2 of the 3 for OIDC 
<img width="473" alt="image" src="https://user-images.githubusercontent.com/7817263/225734591-b6941b6a-b343-4584-aa2e-3f2789feaaa2.png">